exercises/leap/.meta/hints.md

```diff
@@ -0,0 +1,3 @@
+Since this is your first Java exercise, we've included a detailed [TUTORIAL.md](https://github.com/exercism/java/blob/master/exercises/hello-world/TUTORIAL.md) file that guides you through your solution. Check it out for tips and
+assistance!
+
```

Comment:
I would suggest to use the text from [here](https://github.com/exercism/java/blob/master/exercises/two-fer/.meta/hints.md). It describes in detail how to start solving an exercise.

Reply:
Updated.